src/node_main.cc

```diff
@@ -7,7 +7,7 @@ int wmain(int argc, wchar_t *wargv[]) {
   if (!IsWindows7OrGreater()) {
     fprintf(stderr, "This application is only supported on Windows 7, "
                     "Windows Server 2008 R2, or higher.");
-    exit(1);
+    exit(0xd8);     // code: ERROR_EXE_MACHINE_TYPE_MISMATCH
```

Comment:
@addaleax thank for your suggestion, `windows.h` has not been used yet in other sources, such that including windows.h here would increase the building time because:

> It defines a very large number of Windows specific functions that can be used in C.

I'm not sure if we should include that file for a simple macro?